lib/pronto/rubocop.rb

```diff
@@ -60,12 +60,7 @@ def processed_source_for(patch)
     end
 
     def level(severity)
-      case severity
-      when :refactor, :convention
-        :info
-      when :warning, :error, :fatal
```

Comment:
@tlikai I think that `:error` and `:fatal` should continue translating to `:error` and `:fatal`. What do you think?